Library/Formula/tesseract.rb

```diff
@@ -65,6 +65,12 @@ class Tesseract < Formula
   end
 
   def install
+    if build.with? "training-tools"
+      icu4c = Formula["icu4c"]
+      ENV.append "CFLAGS", "-I#{icu4c.include}"
```

Comment:
Mind using `opt.include` and `opt.lib` instead of `include` and `lib` on these two lines? Thanks!